project/ottersome/README.md

```diff
@@ -0,0 +1,186 @@
+# RacImgLib(RIL): General Purpose Image Processing Framework
+
+This is a general purpose framework for image processing tasks. Its aim for the
+course will be humble in purpose but its lifetime will be longer than the
+course. It will start by with the aim of solving 3 main tasks related to image
+processing. This will be stated concretely and tackled generally. Once the
+course is over my aim is to use it for my own photography needs. The tools will 
+be more on the image editing/post-production side rather than something like
+openCV. A little disclaimer will be made: Due to restrictions in time I will
+most likely be able to implement only one or two of these features, but I state
+three in case time allows to cover them all. If I find that one of them doesn't
+interest me or is too lengthy then I might switch them as well.
+
+# Basic Information
+
+[Repository can be found here](https://github.com/ottersome/RIL).
+
+# Problem to Solve
+
+This is mostly a utility that solves problems of incovenience for myself. As
+I want a longer lasting project than a single semester I went with a broader
+subject.
+This is of use to anyone or any industry doing image processing. 
```

Comment:
I am not sure whether or not you intended to use a separate paragraph here.  If you do, add a blank line above.